src/firefly/js/visualize/ui/MouseReadoutOptionPopups.jsx

```diff
@@ -145,20 +169,21 @@ class MouseReadoutOptionDialog extends PureComponent {
 MouseReadoutOptionDialog.propTypes= {
 	groupKey:   PropTypes.string.isRequired,
 	fieldKey:   PropTypes.string.isRequired,
-	radioValue:   PropTypes.string.isRequired
+	radioValue:   PropTypes.string.isRequired,
+    isHiPS: PropTypes.bool.isRequired
 };
 
 
 // ------------ React component
-function CoordinateOptionDialogForm({ groupKey,fieldKey,radioValue}) {
+function CoordinateOptionDialogForm({ groupKey,fieldKey,radioValue, optionList}) {
 
 
 	return (
 
-		<FieldGroup groupKey={groupKey} keepState={true}>
+		<FieldGroup groupKey={groupKey} keepState={false}>
 			<div style={ dialogStyle} onClick={ () => doDispatch(groupKey, fieldKey) } >
 					<div style={leftColumn} title='Please select an option'> Options</div>
```

Comment:
great! I am learning this way (using a <div> around field group component) to dispatch function  for field change.